scripts/LedSignMock.py

```diff
@@ -0,0 +1,37 @@
+import curses
+import argparse
+
+
+
+class LedSignMock(data):
+    parser = argparse.ArgumentParser(description="Prints mock output.")
+    parser.add_argument('--mock-output', help = "Displays Mock Output", )
+    args = parser.parse_args()
+
+    menu = ['   ','SCE LED Sign Visualization ', ' ','Most Recent Health Check: ', 'Most Recent Sign Update: ',' ',
+    'Current Sign Text:', 'Current Brightness:', 'Current Scroll Speed:', 'Current Text Color: ', 'Current Background:',
+    'Current Border Color: ' ]
+    ESCKEY = 27
+    def print_menu(self, stdscr, selected_row_idx):
+        data = "info here"
+        stdscr.clear()
+        stdscr.border(0)
+        height, width = stdscr.getmaxyx()
+        for idx, row in enumerate(menu):
+            x = width//3 - len(menu)
```

Comment:
leave a nice comment here too, pretend its your first time looking at this file
```suggestion
            # calculate the x width blah blah blah
            x = width // 3 - len(menu)
```